node/pkg/logscribe/app.go

```diff
@@ -2,22 +2,67 @@ package logscribe
 
 import (
 	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"net/http"
 	"os"
+	"sort"
 	"time"
 
 	"bisonai.com/orakl/node/pkg/db"
+	errorsentinel "bisonai.com/orakl/node/pkg/error"
 	"bisonai.com/orakl/node/pkg/logscribe/api"
 	"bisonai.com/orakl/node/pkg/logscribe/utils"
 	"github.com/gofiber/fiber/v2"
+	"github.com/google/go-github/github"
+	"github.com/robfig/cron/v3"
 	"github.com/rs/zerolog/log"
+	"golang.org/x/oauth2"
 )
 
 const (
-	logsChannelSize             = 10_000
-	DefaultBulkLogsCopyInterval = 3 * time.Second
+	logsChannelSize              = 10_000
+	dbReadBatchSize              = 1000
+	DefaultBulkLogsCopyInterval  = 30 * time.Second
+	readDeleteLogsQuery          = `DELETE FROM logs RETURNING *;`
+	topOccurrencesForGithubIssue = 5
+	logAlreadyProcessedQuery     = `SELECT COUNT(*) FROM processed_logs WHERE log_hash = @hash`
```

Comment:
it would be better if this can be checked in bulk rather than one by one for each entries.
well I'll keep this optional since we only look for maximum five entries per service so should be less than 100 calls